### DISCUSSION TITLE
As a user, I would like to define more specific opening hours for my Restaurant, Shop, etc.

### DISCUSSION THREAD

**Discussion by pkritzinger** [noi-techpark/it.bz.opendatahub.databrowser#358] [Ideas]
- Define multiple opening hours per day (e.g. Monday | open from 9-12 and 15-19)
- Define different opening hours per day (e.g. Monday | open from 9-19 || Tuesday | open from 9-16)

**Comment by gappc** [comment]
@pkritzinger @sseppi is this issue still relevant?

Furthermore, is the use case supported by ODH? If not, I'd suggest to close this issue here and - if necessary - open a new one on the ODH project, because the Data Browser can not implement functionality on data / structure that does not exist.

**Comment by pkritzinger** [comment]
@gappc from my point of view this is an issue that belongs to the backlog of the backlog :)
We discussed this topic once and wanted to document it, but there's no need to implement it right now. Correct @sseppi ?

**Comment by sseppi** [comment]
@gappc and @pkritzinger 

Right this issue has been created to discuss a possible future enhancement that we shouldn't forget. For this reason the issue for the moment isn't assigned to the Project or any Milestone.

I would suggest to not close it, otherwise we completely forget about this topic.

**Comment by gappc** [comment]
@sseppi @pkritzinger :+1:

**Comment by gappc** [comment]
@sseppi taking a look at all issues I came back to this one. Given that ODH does not support the use case, I think this issue is better converted to a discussion to not pop up in the issues list. In my opinion, issues are something we can take care of. If we just want to have a reminder, its better to put the stuff into discussions. Otherwise we have our issue list full of reminders and need to figure out what real issues are.

If you agree, please convert this issue into a discussion.
